keyboards/anavi/macropad12/keymaps/default/keymap.c

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _BASE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(
+        KC_1   , KC_2   , KC_3   ,
+	KC_4   , KC_5   , KC_6   ,
+	KC_7   , KC_8   , KC_9   ,
+	BL_STEP , RGB_MOD , RGB_TOG
```

Comment:
human readable formatting
```suggestion
        KC_1,    KC_2,    KC_3,
	    KC_4,    KC_5,    KC_6,
    	KC_7,    KC_8,    KC_9,
	    BL_STEP, RGB_MOD, RGB_TOG
```